console/src/modules/Console.rb

```diff
@@ -240,6 +241,26 @@ def Console
       nil
     end
 
+    # Console fonts map
+    #
+    # The map can be read from two different files:
+    #
+    # * `consolefonts_PRODUCT.ycp` where PRODUCT is the downcased product's
+    #   short name. For example, `consolefonts_sles.ycp`.
+    # * `consolefonts.ycp` as a fallback.
```

Comment:
I would prefer to keep the .ycp version for SLE 12 SP2 and CASP. But, what about using YAML in `master`? Does it sound reasonable?